x-pack/plugins/reporting_notifier/constants.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const JOB_COMPLETION_NOTIFICATIONS_SESSION_KEY =
+  'xpack.reporting.jobCompletionNotifications';
+
+export const JOB_COMPLETION_NOTIFICATIONS_POLLER_CONFIG = {
+  jobCompletionNotifier: {
+    interval: 10000,
+    intervalErrorMultiplier: 5
+  } // prettier-ignore
```

Comment:
Curious as to the ignore here?

Reply:
Oops, it isn't necessary :)

It was a leftover from a variant of the code that prettier was formatting in a bad way

Reply:
39abf8ba9b